config/mssql/iam/iam_service.json

```
@@ -1,7 +1,7 @@
 {
   "basePath": "http://localhost:5000/",
   "claimPath": "http://localhost:5000/claims/ensure",
-  "disableClaimCheck": false,
+  "disableClaimCheck": true,
```

In Production Environments sollte die Option hier immer auf "false" gesetzt sein. 

```suggestion
  "disableClaimCheck": false,
```

---

Jau das habe ich vergessen wieder zu deaktivieren 👍